### PR TITLE
dockerized

### PR DESCRIPTION
**Pull request by salarnasiri**
Hi, I've added this project's docker to it, please accept the pull request . Thanks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,7 @@ env/
 
 __pycache__/
 peewee.db
+
+.idea
+.idea/
+docker-compose.yml
```
Comment (franciscod) [inline .gitignore]: is this `docker-compose.yml` a docker generated file?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,7 @@ env/
 
 __pycache__/
 peewee.db
+
+.idea
+.idea/
```
Comment (franciscod) [inline .gitignore]: This has nothing to do with the bot! See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+README.md
+Changelog.md
+Makefile
+docker-compose.yml
+docs
+.ipynb_checkpoints/*
+/notebooks/*
+/unused/*
+Dockerfile
+.DS_Store
```
Comment (franciscod) [inline .dockerignore]: do we need all this?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#Download base image python 3.5
+FROM python:3.5
+
+WORKDIR /bot
+
+COPY ./requirements.txt .
```
Comment (franciscod) [inline Dockerfile]: Isn't this file included in the `COPY . .` a few lines below?

**Comment by franciscod** [comment]
Thanks for the PR! I've left a few comments on stuff that might be improved :)

I'm not really sure if adding/maintaining a Dockerfile in the repo is a good idea. Virtualenv provides good enough isolation already (maybe I'm missing something?). Why docker instead of just virtualenv?

File: .gitignore (hunk 1 above)
Comment (salarnasiri) [reply .gitignore]: No.
i will remove it from gitignore in next update

File: .gitignore (hunk 2 above)
Comment (salarnasiri) [reply .gitignore]: Thanks i didn't know this.
I'm going to fix it soon.

File: .dockerignore (hunk 3 above)
Comment (salarnasiri) [reply .dockerignore]: I will remove unnecessary files. You are right

**Comment by salarnasiri** [comment]
@franciscod please check again and merge PR.
tanQ

**Comment by salarnasiri** [comment]
@franciscod yuhuu

**Comment by franciscod** [comment]
@salarnasiri okay, this looks useful! Please resolve the conflicts on the README

**Comment by franciscod** [comment]
I've added a minimalistic Dockerfile:
https://github.com/franciscod/telegram-twitter-forwarder-bot/commit/7b82aa162b28ae8cc10b6ffce6b5e474ed446119

**Pull request closed without merging**